scripts/docker/docker-pull-async.ps1

```diff
@@ -12,5 +12,3 @@ function DockerPull {
   $results
 }
 
-DockerPull microsoft/windowsservercore
```

Comment:
My use case is to have these both Docker images baked into the VM. Of course it is much bigger, but I'm gaining speed as I often destroy the Vagrant box and rebuild it for demo scenarios. Using VirtualBox, VMware Fusion/Workstation Pro or Hyper-V with Vagrant can use linked clones. 

So spinning up a Vagrant box is only a few seconds and the fast VM image needs disk space only once on the host. My fully installed Docker engine is ready in less than a minute and without pulling the big servercore base image. So I can start building Windows Docker images as fast as spinning up a Linux docker machine.

And when a new set of base images is available I usually run another packer build and replace that Vagrant box as it also now has the latest Windows updates installed as well.